cli/command/command.go

```diff
@@ -89,10 +89,23 @@ func ConvertCommand() cli.Command {
 func UpCommand() cli.Command {
 	return cli.Command{
 		Name:  "up",
-		Usage: "Submit rc, svc objects to kubernetes API endpoint",
+		Usage: "Deploy your Dockerized application to Kubernetes (by default create dc and svc)",
 		Action: func(c *cli.Context) {
 			app.Up(c)
 		},
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:   "file,f",
```

Comment:
We can set both `-f` and `--bundle` flags in a function and call that function in kompose up and kompose convert